### PR TITLE
feat: show certified copy text in line item

### PR DESCRIPTION
**Pull request by jgbenito7**
Shows text for line item indicating it is a certified copy.

https://kofile.tpondemand.com/entity/52408-ui-in-receipt-view-show-certified

### CODE REVIEW

File: __fixtures__/Support/order-receipt.fixture.js [hunk 1]
```diff
@@ -36,7 +36,8 @@ export default {
         checkbox: {
           isChecked: false,
           onClick: console.log
-        }
+        },
+        isCertifiedCopy: true
```
Comment (EthanJStark) [inline __fixtures__/Support/order-receipt.fixture.js]: Examining the fixtures in the browser, I'm not seeing any certified copy text. Any idea why that'd be?

Comment (jgbenito7) [reply __fixtures__/Support/order-receipt.fixture.js]: did you build?

Comment (EthanJStark) [reply __fixtures__/Support/order-receipt.fixture.js]: Yeah, building helps.

**Pull request merged**